addon/index.js

```diff
@@ -34,9 +38,13 @@ export default function (app, prefix) {
   for (var i = 0; i < moduleNames.length; i++) {
     var moduleName = moduleNames[i];
     if (moduleName.lastIndexOf(initializerPrefix, 0) === 0) {
-      initializers.push(moduleName);
+      if (!_endsWith(moduleName, '-test')) {
+        initializers.push(moduleName);
+      }
     } else if (moduleName.lastIndexOf(instanceInitializerPrefix, 0) === 0) {
-      instanceInitializers.push(moduleName);
+      if (!_endsWith(moduleName, '-test')) {
+        instanceInitializers.push(moduleName);
+      }
```

Comment:
without these changes, the app fails to boot with:

<img width="1413" alt="screen shot 2018-04-16 at 14 26 54" src="https://user-images.githubusercontent.com/2526/38811766-635fd1cc-4182-11e8-9a27-5690307b8487.png">